Examples/ExamplesApp/Examples.swift

```diff
@@ -15,98 +15,88 @@
-                        .navigationBarTitleDisplayMode(.inline)
                     }
-                    .isDetailLink(false)
+                case .example(let example):
+                    Text(example.name)
+                        .tag(example)
                 }
-                .navigationTitle("Toolkit Examples")
             }
+            .navigationTitle("Toolkit Examples")
         } detail: {
-            NavigationStack {
-                if let example = selectedExample {
-                    example.view
-                        .navigationTitle(example.name)
-                        .navigationBarTitleDisplayMode(.inline)
-                } else if selectedCategory != nil {
-                    Text("Select an example")
-                } else {
-                    Text("Select a category")
-                }
+            if let selectedExample {
+                selectedExample.view
+                    .navigationTitle(selectedExample.name)
+                    .navigationBarTitleDisplayMode(.inline)
+            } else {
+                Text("Select an example")
             }
         }
+        // visionOS doesn't provide the column visibility toggle like
+        // iPadOS and Mac Catalyst so conditionally hide the column.
+#if !os(visionOS)
+        .onChange(selectedExample) { _ in
+            columnVisibility = .detailOnly
+        }
+#endif
     }
     
-    static func makeCategories() -> [Category] {
-        let common: [Category] = [
-            .geoview,
-            .views
-        ]
+    static func makeCategories() -> [ListItem] {
 #if os(iOS) && !targetEnvironment(macCatalyst)
-        return [.augmentedReality] + common
+        return [.category("Augmented Reality", examples: .augmentedReality)]
 #else
-        return common
+        return []
 #endif
     }
+    
+    static func makeListItems() -> [ListItem] {
+        (makeCategories() + makeUncategorizedExamples())
+            .sorted(by: { $0.name < $1.name })
+    }
+    
+    static func makeUncategorizedExamples() -> [ListItem] {
+        return [
+            .example("Basemap Gallery", content: BasemapGalleryExampleView()),
+            .example("Bookmarks", content: BookmarksExampleView()),
+            .example("Compass", content: CompassExampleView()),
+            .example("Feature Form", content: FeatureFormExampleView()),
+            .example("Floating Panel", content: FloatingPanelExampleView()),
+            .example("Floor Filter", content: FloorFilterExampleView()),
+            .example("Overview Map", content: OverviewMapExampleView()),
+            .example("Popup", content: PopupExampleView()),
+            .example("Scalebar", content: ScalebarExampleView()),
+            .example("Search", content: SearchExampleView()),
+            .example("Utility Network Trace", content: UtilityNetworkTraceExampleView())
+        ]
+    }
 }
 
 @MainActor
-extension Category {
+extension Array where Element == Example {
```

Comment:
Instead of extending `Array`, just extend `ListItem`:

```swift
#if os(iOS) && !targetEnvironment(macCatalyst)
extension Examples.ListItem {
    static var augmentedRealityCategory: Self {
        .category(
            "Augmented Reality",
            examples: [
                Example("Flyover", content: FlyoverExampleView()),
                Example("Tabletop", content: TableTopExampleView()),
                Example("World Scale", content: WorldScaleExampleView())
            ]
        )
    }
}
#endif
```